web_src/css/repo.css

```diff
@@ -3080,7 +3080,7 @@ tbody.commit-list {
 }
 
 .commit-status-list {
-  max-height: 195px; /* fit exactly 4 items */
+  max-height: 196px; /* fit exactly 4 items */
```

Comment:
I guess the true value must lie somewhere in between :D

Reply:
Also I think we should display 6 checks and reduce the padding on each a bit.

Reply:
Tweak done in https://github.com/go-gitea/gitea/pull/27932/commits/0a70fcc5d22483f8a9722b359568a6e1933c4c76:

<img width="517" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/93659f2d-f060-4d62-bfcd-404c08c73560">